### ISSUE TITLE
Possible to use OBS Virtual Webcam as input without capture frame by frame method? 

### ISSUE THREAD

**Issue by ikmal-asraf** [Deci-AI/super-gradients#1967]
### 💡 Your Question

Trying to test the model performance in real time .Since it is hard to setup the environment to test it real time , virtual webcam is use to solve it . OpenCV can detect it , by just using

cap = cv2.VideoCapture(1)

for YOLO 

model = models.get('yolo_nas_s',pretrained_weights="coco").to(device)
model.predict_webcam()

but it does not detect the virtual camera I setup , is there simple solution rathen then try to use the ret , capture frame by frame method?

**Comment by BloodAxe** [comment]
For `predict_webcam` we use `cv2.CAP_ANY` as a default device, meaning `find first compatible device`.
Currently there is not API to explicitly choose what capture device to use. Sorry about that :(

I want to stress that testing any model with pytorch runtime is a wrong way to assess model's performance. If your task demands real-time performance you definitely want to export the model to ONNX and run using specialized inference software like TensorRT / OpenVINO / ONNXRuntime.
Otherwise you would get very wrong, misleading numbers.
